### ISSUE TITLE
Rename Information Button in Tagebuch page 

### ISSUE THREAD

**Issue by DerVogel2020** [corona-warn-app/cwa-app-android#2944]
## Avoid duplicates
* [x] This enhancement request has not already been raised before
* [x] Enhancement request is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in [CWA-Wishlist](https://github.com/corona-warn-app/cwa-wishlist)
* [ ] If you are proposing a new feature, please do so in [CWA-Wishlist](https://github.com/corona-warn-app/cwa-wishlist)

## Current Implementation

Currently it says Information when you press the 3 dot button in the Tagebuch page.

## Suggested Enhancement

Rename to "Informationen " as it is in the Check in page.

----
Internal Tracking-ID: [EXPOSUREAPP-6762](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6762)

**Comment by DerVogel2020** [comment]
See #2543 in corona-warn-app-iOS .

**Comment by Ein-Tim** [comment]
FYI: 
You can just post the URL and will be shown like this: https://github.com/corona-warn-app/cwa-app-ios/issues/2543

**Comment by MikeMcC399** [comment]
@DerVogel2020 

To copy a link from any posting, click on the three dots to the right of the smiley and select "Copy link" from the drop-down menu.

You can get a link to the top level of a post or to an individual comment that way. 

You could also correct your posting with the wrong link by editing your post.

**Comment by DerVogel2020** [comment]
> @DerVogel2020 
> 
> To copy a link from any posting, click on the three dots to the right of the smiley and select "Copy link" from the drop-down menu.
In the android app there is no such option.


> You could also correct your posting with the wrong link by editing your post.
I'll do that.

**Comment by MikeMcC399** [comment]
@DerVogel2020

>> To copy a link from any posting, click on the three dots to the right of the smiley and select "Copy link" from the drop-down menu.
>
> In the android app there is no such option.

That's funny, because I didn't even know there was a GitHub app for Android! I have always used the web interface with the Google Chrome browser. So thanks for introducing me to that option!

I took a look at the app, and on Android the option "Share" will get you a link to an issue, but I found that it is not so convenient to use as in the web interface.
